indexer-app/resources/schemas/dataqualitysummary/v1.0.0/schema.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
this comment is for all the schemas, if these are going to be checked into the schema repository, and those files will include a metadata example, then we don't need one here, however, having a metadata example for each schema makes it easy to do a quick curl to load a record locally.